### PR TITLE
Fix issue 354 on recreate session on Bing & Inform user that the server is running

### PR DESCRIPTION
**Pull request by vuminhquang**

### CODE REVIEW

**Comment by vuminhquang** [comment]
Fix for issue #348

**Comment by liguanghe** [comment]
hi, i changed the file as you do. then $node bin/server.js
shows: 
{"message":"Route GET:/ not found","error":"Not Found","statusCode":404}
how could i fix it~? thanks

**Comment by vuminhquang** [comment]
You can GET ping
server.get('/ping', () => Date.now().toString());

**Comment by liguanghe** [comment]
thanks a lot for repeating,
i'm new to javascript and node.js, i'm not sure how to get ping ,'server.get('/ping', () => Date.now().toString());' is in the code.what else i should do~?

**Comment by vuminhquang** [comment]
you can test the server by localhost:3000/ping (just read this line in 'server.get('/ping', () => Date.now().toString());' bin/server.js)

**Comment by vuminhquang** [comment]
Fix for issue #354

**Comment by liguanghe** [comment]
visit http://127.0.0.1:3000/ping,show numbers. seems this part is work.
visit http://127.0.0.1:3000/conversation,show:{"message":"Route GET:/conversation not found","error":"Not Found","statusCode":404} so,how could i fix it~?thanks.

**Comment by liguanghe** [comment]
i asked AI, the answer:
As for the error message your colleague is receiving, it seems that the server is unable to find a route for GET requests to '/conversation'. This is likely because the server is only configured to handle POST requests to this endpoint. Your colleague should ensure that they are sending a POST request to '/conversation' instead of a GET request. 
how could i fix it? thanks~

File: src/BingAIClient.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ export default class BingAIClient {
             onProgress = () => { };
         }
 
-        if (jailbreakConversationId || !conversationSignature || !conversationId || !clientId) {
```
Comment (waylaidwanderer) [inline src/BingAIClient.js:183]: This is not how the logic is intended to work. `jailbreakConversationId` creates a new conversation on purpose every message and restores the message history by sending the locally stored messages as context to Bing.

File: bin/server.js [hunk 2]
```diff
@@ -159,13 +159,21 @@ server.post('/conversation', async (request, reply) => {
...
+    console.log(`Server is listening on ${address}`)
 });
 
+// // wait for the server to start
+// await new Promise(resolve => server.ready(resolve));
+
+// fastify.on('listening', (address) => {
+//     console.log(`Server listening on ${address}`);
+// });
+
```
Comment (waylaidwanderer) [inline bin/server.js:176]: You should delete code rather than comment it out, since they can be restored via git if still needed in the future.

**Comment by waylaidwanderer** [comment]
Sorry, not sure I really see a point in the PR. Plus it also does two separate things. Each PR should only deal with one feature so if one of the features has an issue, it doesn't impact the other features from being merged. Closing for now.

File: bin/server.js (hunk 2 above)
Comment (liguanghe) [reply bin/server.js:176]: so~what should i do~?

**Pull request closed without merging**